packages/remix-server-runtime/__tests__/injectServerEffect-test.ts

```diff
@@ -0,0 +1,30 @@
+import {
+  injectServerEffect,
+  SERVER_EFFECTS_KEY,
+  ServerEffect,
+  getGlobalEffectById,
+} from "../injectServerEffect";
+
+describe(injectServerEffect, () => {
```

Comment:
```suggestion
describe("injectServerEffect", () => {
```

Reply:
Can you elaborate more on why repeat the function name when you can infer it by passing the function reference to the `describe` block? 

The code I pushed yields `describe(injectServerEffect.name)`, which turns into your suggestion exactly.

Reply:
My suggestion is how we do it in the rest of the codebase, so I just want to keep it consistent